static/bidder-info/richaudience.yaml

```diff
@@ -0,0 +1,15 @@
+maintainer:
+  email: partnerintegrations@richaudience.com
+gvlVendorID: 108
+gdpr:
+  timeouts_ms:
+    init_vendorlist_fetches: 5000
+    active_vendorlist_fetch: 5000
+modifyingVastXmlAllowed: true
+capabilities:
+  site:
+    mediaTypes:
+      - banner
+userSync:
+  iframe:
+    url: "https://sync.richaudience.com/74889303289e27f327ad0c6de7be7264/?consentString={{.GDPRConsent}}&r={{.RedirectURL}}[PDID]"
```

Comment:
What is the purpose of `[PDID]` at the end of the url? This looks like it will cause the redirect url to be incorrectly formed.

Reply:
in next commit i have solved

For us this macro is correct, i will not change this

Reply:
This is not the correct way to specify the macro. It only works due to coincidental positioning of the fields of the default redirect url template . Please see: https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#bidder-info

Please remove `[PDID]` from the url and add it as a userMacro config:
```
url: "https://sync.richaudience.com/74889303289e27f327ad0c6de7be7264/?consentString={{.GDPRConsent}}&r={{.RedirectURL}}"
userMacro: "[PDID]"
```